Tests/GRDBTests/FoundationNSURLTests.swift

```diff
@@ -4,6 +4,9 @@ import GRDB
 class FoundationNSURLTests: GRDBTestCase {
     
     func testNSURLDatabaseRoundTrip() throws {
+        #if !canImport(ObjectiveC)
+        throw XCTSkip("NSURL unavailable")
```

Comment:
`NSURL` is less useful than `NSNumber`/`NSDecimalNumber`, so here I'll just recommend that the whole test class is disabled on platforms that do not support NSURL.